dht-node/src/dht_node/index.ts

```diff
@@ -179,7 +186,7 @@ export const startDHT = async (topic: string): Promise<void> => {
   }
 }
 
-async function writeFederatedHomeCommunityEnries(pubKey: any): Promise<CommunityApi[]> {
+export async function writeFederatedHomeCommunityEntries(pubKey: string): Promise<CommunityApi[]> {
```

Comment:
No export needed anymore, as the unit tests do not import anymore

Reply:
done